docs/install_guides/pypi-openvino-dev.md

```diff
@@ -69,14 +69,15 @@ There are two options to install OpenVINO Development Tools:
 
 #### Installing Default Components
 
-To install the default components in the package (see the [What's in the Package](#whats-in-the-package) section of this article), use the following command:
+To install OpenVINO Development Tools (see the [What's in the Package](#whats-in-the-package) section of this article) into an existing environment with the deep learning framework used for the model training, run the following command:
```

Comment:
This change makes it sound like you need a deep learning framework used for model training. This is one usecase, but definitely not the only one, for openvino-dev. You may have an .onnx model, not even know what the source framework was. No need for a source training framework to use benchmark_app or MO for ONNX models. You may also have a ready made IR model and just want to use benchmark_app.

Reply:
Here I mean to use the exact same framework used for the model training. If you don't train the model that you are going to convert, then use extras.

Reply:
True, but that's not documented anymore. This section used to say that, to install just the tools, do `pip install openvino-dev`, without indicating that you needed a framework. But now that same section says to do this "if you already have an environment with a framework". 

It's also a bit confusing that step 1, 2 and 3 are about creating a fresh environment, and that now in step 4 we talk about "an existing environment" that is separate from the one we just created in the previous steps. It may be clearer to move this to a separate section. For example a "quick install" at the top of the document. If you already have an environment and want to install openvino-dev, do `pip install openvino-dev`. For more detailed instructions on how to install openvino-dev in its own environment, see instructions below"

Reply:
@helena-intel, it is great catch about confusing step 1,2,3. They are only applicable for `extras` and not needed for the default installation. I will correct it.

Reply:
@helena-intel, I removed this confusion. Now we have two clear ways for installation with description about when to apply each of them.
Please review it again.